delta/introduction.adoc

```diff
@@ -1,23 +1,36 @@
 [[introduction]]
 == Introduction
-Delta API assumes a bidirectional communication channel.
-The _client_ initiates the channel with the _repository_ to open a _participation_{fn-org337}.
+
+Definitions:
+
+* A <<repository>> is a set of partitions (effectively comprising a "`forest`" of LionWeb nodes), all of which MUST have unique IDs.
```

Comment:
I would say the repository is not just the data, but also the organization of such data and all the supporting structures (like indexes, structures to track assigned IDs, also support for the _trash_, etc.)